graphstorm-processing/tests/test_dist_heterogenous_loader.py

```diff
@@ -1033,3 +1033,119 @@ def test_edge_custom_label(spark, dghl_loader: DistHeterogeneousGraphLoader, tmp
     assert train_total_ones == 3
     assert val_total_ones == 3
     assert test_total_ones == 3
+
+
+def test_node_custom_label_multitask(spark, dghl_loader: DistHeterogeneousGraphLoader, tmp_path):
+    """Test using custom label splits for nodes"""
+    data = [(i,) for i in range(1, 11)]
+
+    # Create DataFrame
+    nodes_df = spark.createDataFrame(data, ["orig"])
+
+    train_df = spark.createDataFrame([(i,) for i in range(1, 6)], ["mask_id"])
+    val_df = spark.createDataFrame([(i,) for i in range(6, 9)], ["mask_id"])
+    test_df = spark.createDataFrame([(i,) for i in range(9, 11)], ["mask_id"])
+
+    train_df.repartition(1).write.parquet(f"{tmp_path}/train.parquet")
+    val_df.repartition(1).write.parquet(f"{tmp_path}/val.parquet")
+    test_df.repartition(1).write.parquet(f"{tmp_path}/test.parquet")
+    class_mask_names = [
+        f"custom_split_train_mask",
+        f"custom_split_val_mask",
+        f"custom_split_test_mask",
+    ]
+    config_dict = {
+        "column": "orig",
+        "type": "classification",
+        "split_rate": {"train": 0.8, "val": 0.1, "test": 0.1},
```

Comment:
Should probably print a warning when both split_rate and custom filenames are provided, noting we'll use the custom files and not do a random split.

Reply:
I can add a comment here.

Reply:
I mean during the parsing of the config, the expectation is to have only one of these two, so should print a warning or error out if we don't want to guess the user's intent.

Reply:
I thought we have done that before. I just check our code again but did not find relative code. I will add it.